src/index.html

```diff
@@ -7,6 +7,361 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo"
+      >
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__button is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              СМАРТФОНЫ И ТЕЛЕФОНЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              ГАДЖЕТЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              ПЛАНШЕТЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button"
+            >
+              ФОТО
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__button nav__button-last"
+            >
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt=""
+          class="card__img"
+        >
+
+        <p class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__item">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
```

Comment:
було б добре вказати модивікатор наприклад (stars--4), щоб можна було регулювати кількість активних зірок
```suggestion
        <div class="card__item">
          <div class="card__stars">
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
          </div>
```